src/editors/_shared/hooks/DataTypes/types/AnyType.test.js

```diff
@@ -0,0 +1,16 @@
+import AnyType from "./AnyType";
+import { testValidation, testStringInput } from "../testUtils";
+
+testValidation(AnyType, [
+  true, true, true, true, true, true, true,
+  true, true, true, true, true, true,
+  true, true, true, true, true, true, true,
+  true, true, true, true, true, true,
+  // onlyStrings
+  true, true, true, true, true, true, true,
+  true, true, true, true, true, true,
+  true, true, true, true, true, true, true,
+  true, true, true, true, true, true, true
+], "[1]", "[1]");
```

Comment:
I think I understand what you are doing here, but this is a bit unbearable to actually know what to see when some test fails...
Maybe you can make the validation suit return an object with a small description of the test like `testValidObject1`, `testValidString1`, `testInvalidString1`, stuff like this. This way we know, when something fails, where it failed and why. Instead of having to compare an array of returns that we don't know by heart...

Reply:
We know where things fail much better than if it were inplace, actually, because the tests tell you the index. Believe me, I've tried. We don't have to know by heart, we can look at testUtils